packages/lodestar/src/tasks/tasks/interopSubnetsJoiningTask.ts

```diff
@@ -0,0 +1,58 @@
+import {ITask} from "../interface";
+import {INetwork} from "../../network";
+import {IBeaconConfig} from "@chainsafe/eth2.0-config";
+import {ATTESTATION_SUBNET_COUNT} from "../../constants";
+import {randBetween} from "@chainsafe/eth2.0-utils";
+
+export interface IInteropSubnetsJoiningModules {
+  network: INetwork;
+}
+
+export class InteropSubnetsJoiningTask implements ITask {
+
+  private readonly config: IBeaconConfig;
+  private readonly network: INetwork;
+
+  public constructor(config: IBeaconConfig, modules: IInteropSubnetsJoiningModules) {
+    this.config = config;
+    this.network = modules.network;
+  }
+
+  public async run(): Promise<void> {
+    for (let i = 0; i < this.config.params.RANDOM_SUBNETS_PER_VALIDATOR; i++) {
```

Comment:
can you show me `RANDOM_SUBNETS_PER_VALIDATOR` in the spec or any spec PR? I can't find it in https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/p2p-interface.md

Reply:
https://github.com/ethereum/eth2.0-specs/blob/v0.9.4/specs/validator/0_beacon-chain-validator.md#phase-0-attestation-subnet-stability